src/open_samus_returns_rando/files/levels/s040_area4.lua

```diff
@@ -207,16 +207,21 @@ function s040_area4.OnEnter_Gamma_001_Dead()
     Game.SetSubAreaCurrentSetup("collision_camera_012", "PostGamma_001", true)
     Game.SetSubAreaCurrentSetup("collision_camera_013", "PostGamma_001", true)
     -- if Game.GetEntity("SpawnGroup019") ~= nil then
-      -- Game.GetEntity("SpawnGroup019").SPAWNGROUP:EnableSpawnGroup()
+    --   Game.GetEntity("SpawnGroup019").SPAWNGROUP:EnableSpawnGroup()
```

Comment:
The whole thing can be reverted as we need to reload the whole room anyways. 
Here and in the `OnSubAreaChange`.